config/settings.yml

```diff
@@ -51,6 +51,38 @@ const cloudProviders = {
   }
 };
 const cloudProvider = cloudProviders.openstack;
+
+const virtualHostCloudProviders = {
```

Comment:
duplicated configs. See if the config `cloudProviders` can be reused here.

Reply:
config cloudProviders can be reused if we create a new container with same credentials as of container for service-fabrik-broker.
We were planning to reuse the provider container specific for rabbitmq service that already exists in landscapes. Other services use it for backup metadata purposes but because for RabbitMQ there is no such use case we will reuse it to store virtual host instances metadata.
Credentials for this container are different than the credentials for service-fabrik broker container.
What do you suggest?

Reply:
Ok you can keep seperate config since its different from broker.